src/utils/bluetoothPermissions.ts

```diff
@@ -15,7 +15,8 @@ import {
  * Shows an alert if device's bluetooth is powered off
  */
 export const showBluetoothPoweredOffAlert = async () => {
-  await Alert.alert(i18n.t(i18n.l.bluetooth.powered_off_alert.title), i18n.t(i18n.l.bluetooth.powered_off_alert.message), [
+  // Need to use Linking here - Phone permissions asked for in an alert
```

Comment:
don't think this needs to be replaced. This will open a native dialog which is what we want.